src/pages/Main/index.tsx

```diff
@@ -59,10 +63,19 @@ const MainPage = () => {
       searchString:
         searchString && searchString.length > 0 ? prettifiedBlockSearchString : undefined
     }).catch((errMsg) => console.error(errMsg));
+
     fetchMoreTransfers({
       searchString
     }).catch((errMsg) => console.error(errMsg));
-  }, [fetchMoreTransfers, fetchMoreBlocks, searchString, navigate]);
+
+    fetchMoreCollections({
+      searchString
+    }).catch((errMsg) => console.error(errMsg));
```

Comment:
doesn't seem like a proper handler, for now - I would rather fall with the exception rather than just silently ignore it